scss.scss

```diff
@@ -1,5 +1,3 @@
-// stylelint-disable property-blacklist, scss/dollar-variable-default
```

Comment:
@XhmikosR, I added these because they are required by Bootstrap. I don't want to maintain different files for Bootstrap and RFS.

Reply:
Oh, I didn't think of that. I'll revert the scss changes only then.